src/difference.js

```diff
@@ -4,7 +4,7 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Finds the set (i.e. no duplicates) of all elements in the first list not
- * contained in the second list.
+ * contained in the second list (in terms of R.contains).
```

Comment:
Hrmm, [R.contains](http://ramdajs.com/0.22.1/docs/#contains) mentions that it operates in terms of R.equals. I think it's beneficial to clarify the behaviour for when there are objects involved. Otherwise the behaviour is undefined on lists, because if it uses `indexOf` then:
```javascript
R.difference([{1:2}], [{1:2}]) //=> [{1:2}]
```

Since testing membership with `indexOf` or `==` does reference equality it's tough to know what would happen.

Clarifying that we use R.equals or R.contains explains the actual behaviour:
```javascript
R.difference([{1:2}], [{1:2}]) //=> []
```